### PR TITLE
Add routing

### LINKED ISSUE #2: Add routing
TBD:
Search page is main if no location entered.
Created pages should be just with name of the page 
Add routes: 
1. search
2. forecast
3. details

### PR DESCRIPTION
**Pull request by Aleshchenk0**

### CODE REVIEW

File: src/routes/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import Search from '../pages/Search.svelte';
```
Comment (Avareco) [inline src/routes/index.js]: According to Structure 
image
 components should be in `routes` folder same as index.js for now I don't see a sense to separate them. @oleksandrstarov What do you think?

**Comment by Avareco** [comment]
Is any ability to get rig of the #
image

File: src/App.svelte [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
+  import { routes } from './routes/index.js';
   import { Badge } from 'flowbite-svelte';
 </script>
 
+<Router {routes} />
+
 <main>
   <Badge>App</Badge>
-  <img src={svelteLogo} alt="Svelte Logo" />
+  <img src="{svelteLogo}" alt="Svelte Logo" />
```
Comment (oleksandrstarov) [inline src/App.svelte]: Do we need to parentheses  brackets here?

**Comment by Avareco** [comment]
Why do we need date on the details page
image

**Review by oleksandrstarov — COMMENTED** [review]
Looks good, there might be few points regarding routes names, but we can take it on our Monday meeting

File: src/App.svelte (hunk 2 above)
Comment (Avareco) [reply src/App.svelte]: > Do we need to parentheses brackets here?

Its eslint issue will be fixed in the my next PR

**Review by Avareco — CHANGES_REQUESTED** [review]

**Comment by Aleshchenk0** [comment]
> Is any ability to get rig of the # image

I'll check it

**Comment by Aleshchenk0** [comment]
> Why do we need date on the details page image

This redundant field, I'll remove it, thanks.

File: src/routes/index.js (hunk 1 above)
Comment (Aleshchenk0) [reply src/routes/index.js]: Agree fixed it

**Review by Avareco — APPROVED** [review]
LGTM

**Review by oleksandrstarov — APPROVED** [review]
Good job

**Pull request merged**